genstudio-mlr-claims-app/src/genstudiopem/web-src/src/components/RightPanel.tsx

```diff
@@ -94,13 +89,12 @@ export default function RightPanel(): JSX.Element {
     }
   };
 
-  const runClaimsCheck = async (
-    experience: Experience,
-    selectedClaimLibrary: any
-  ): Promise<void> => {
+  const runClaimsCheck = async (experience: Experience): Promise<void> => {
     setIsLoading(true);
     try {
-      const result = validateClaims(experience, selectedClaimLibrary);
+      // run all claim libraries
+      const allClaimLibraries = TEST_CLAIMS.map((library) => library.id);
+      const result = validateClaims(experience, allClaimLibraries);
       // Add a minimum loading time of 0.5 seconds
       await new Promise((resolve) => setTimeout(resolve, 500));
```

Comment:
now that this will happen on open, lets remove the artificial pause. the shorter the time it takes to run, the better.